src/Caches/Factory.php

```diff
@@ -0,0 +1,21 @@
+<?php
+declare(strict_types=1);
```

Comment:
here strict types is used but as far as i know it is missing in multiple other files, how is it supposed to be?

Reply:
with strict types, that is an oversight if missing